scalableminds/webknossos
# [Skeletons] Allow deletion of root group

There is no quick way to delete all skeleton groups and trees. Deleting the `Root` group would do that but has been disabled up until point. This PR changes that and allows the Root group and all it's children (aka everything) to be deleted.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open annotation with sub-groups and trees
- Right-click on `Root` group
- Select `Delete Group`
- Everything should be gone. The whole Skeleton tab should be empty. A Root node will be automatically re-created when placing a new node.

### Issues:
- fixes #5773

------
(Please delete unneeded items, merge only when none are left open)
